DirectProgramming/C++SYCL/MapReduce/guided_MonteCarloMultiGPU_SYCLMigration/sample.json

```diff
@@ -14,9 +14,13 @@
 			"mkdir build",
       		        "cd build",
            		"cmake ..",
+			"cmake -D INTEL_MAX_GPU=1 ..",
+			"cmake -D NVIDIA_GPU=1 ..",
            		"make",
 			"make run_sm_cpu",
-			"make run_sm_gpu"
+			"make run_sm_gpu",
+			"make run_sm_gpu_opencl"

```

Comment:
@AvijitBag07 Forgot to add ",".  ``"make run_sm_gpu_opencl",``

Reply:
Added the comma